compiler/luci/import/include/luci/Import/TensorBuilder.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_IMPORT_TENSOR_BUILDER_H__
+#define __LUCI_IMPORT_TENSOR_BUILDER_H__
+
+#include "GraphBuilderContext.h"
+#include "GraphBuilderBase.h"
+
+#include <mio/circle/schema_generated.h>
+
+namespace luci
+{
+
+/**
+ * @brief Tensor types which requires separated node
+ */
+enum class TensorBuilderType
+{
+  INPUT,
+  BUFFER,
+  OUTPUT,
+  // TODO Extend this struct here if needed to add new type of TensorBuilder
+  COUNT // Count of different types
+};
+
+/**
+ * @brief Creates nodes from given Tensor and context
+ */
+class TensorBuilder
```

Comment:
I have concerns related to the name of this entity, because it builds nodes (not tensors :slightly_frowning_face: )

Reply:
I feel this so. However, I have not thought of a better name. I'm happy to hear any suggestions about naming!

Reply:
How about `SpecialNodeBuilder`?

Reply:
Renamed as `SpecialNodeBuilder` in current revision.

Reply:
Another thing that came to mind is `EndPointNodeBuilder`